administrator/components/com_cookiemanager/config.xml

```diff
@@ -8,7 +8,10 @@
 			name="policylink"
 			type="menuitem"
 			label="COM_COOKIEMANAGER_POLICY_LINK_LABEL"
-		/>
+			default="100"
```

Comment:
I do not think a value of 100 is a good choice because if I have a menu item ID of 100 there is a conflict. Can the default not just be empty? The default will just be `default=""` and the option value will be empty as well.

Reply:
done